protected/controllers/PatientController.php

```diff
@@ -1981,20 +1981,4 @@ public function actionPreviousElements($element_type_id, $patient_id, $limit = n
         echo CJSON::encode($result);
     }
 
-    public function actionFindDuplicates($firstName, $last_name, $dob, $id = null)
```

Comment:
Why is actionFindDuplicates being removed?

Reply:
It got duplicated due to rebase issues I thought I'd already addressed.

Reply:
Fair enough, quite a few of my recent PRs do the same thing